.nvmrc

```diff
@@ -1 +1 @@
-14.17.0
+14.17.0
```

Comment:
revert this EOF

Reply:
Are you talking about the space? I think it's a prettier thing.

Reply:
Reverted!